pkg/util/validation/validate.go

```diff
@@ -52,6 +52,9 @@ const (
 	exemplarLabelsTooLong    = "exemplar_labels_too_long"
 	exemplarTimestampInvalid = "exemplar_timestamp_invalid"
 
+	// Native Histogram specific validation reasons
+	nativeHistogramBucketsExceeded = "native_histogram_buckets_exceeded"
```

Comment:
Can we make the error more specific like `nativeHistogramBucketCountLimitExceeded`?

Reply:
Addressed in latest commit.